doc/04-context.md

```diff
@@ -0,0 +1,80 @@
+## Context
+
+For every command that castor execute, it uses a context object. This object contains the default value for the `exec` 
+function (directory, environment variables, pty, tty, etc...).
```

Comment:
```suggestion
For every command that castor execute, it uses a context object. This object contains the default value for the `exec`  or `watch`
functions (directory, environment variables, pty, tty, etc...).
```

Reply:
I'm not sure we should mention the watch function here, it's usage should be quite rare